apps/cowswap-frontend/src/legacy/state/price/updater.ts

```diff
@@ -282,5 +363,29 @@ export default function FeesUpdater(): null {
     verifiedQuotesEnabled,
   ])
 
-  return null
+  return quoteParams
+}
+
+function sameQuoteParams(quoteParams1?: FeeQuoteParams, quoteParams2?: FeeQuoteParams) {
+  if (quoteParams1 === undefined) {
+    return quoteParams2 === undefined
+  }
+  if (quoteParams2 === undefined) {
+    return false
+  }
+
+  return (
+    quoteParams1.amount === quoteParams2.amount &&
+    quoteParams1.buyToken === quoteParams2.buyToken &&
+    quoteParams1.chainId === quoteParams2.chainId &&
+    quoteParams1.fromDecimals === quoteParams2.fromDecimals &&
+    quoteParams1.isBestQuote === quoteParams2.isBestQuote &&
+    quoteParams1.isEthFlow === quoteParams2.isEthFlow &&
+    quoteParams1.kind === quoteParams2.kind &&
+    quoteParams1.priceQuality === quoteParams2.priceQuality &&
+    quoteParams1.receiver === quoteParams2.receiver &&
+    quoteParams1.sellToken === quoteParams2.sellToken &&
+    quoteParams1.toDecimals === quoteParams2.toDecimals &&
+    quoteParams1.userAddress === quoteParams2.userAddress
```

Comment:
(I'm lazy to check myself)

Are all of those property primitive types?
For example, is `amount` a string or a CurrencyAmount/BigNumberIsh instance?

Reply:
All primitive types :)